set.go

```diff
@@ -172,7 +173,7 @@ func (s *SetElement) decode() func(b []byte) error {
 				flags := ad.Uint32()
 				s.IntervalEnd = (flags & unix.NFT_SET_ELEM_INTERVAL_END) != 0
 			case unix.NFTA_SET_ELEM_TIMEOUT:
-				s.Timeout = ad.Uint64()
+				s.Timeout = time.Duration(ad.Uint64() * 1000)
```

Comment:
Here and elsewhere: instead of multiplying with 1000, use e.g. time.Millisecond constants to express the conversion you’re doing.

Reply:
Done